docs/migrating.md

```diff
@@ -1 +1,87 @@
 # Migrating
+
+## Configuration
+
+This section outlines configuration settings available in
+[SignalFx Instrumentation for .NET](https://github.com/signalfx/signalfx-dotnet-tracing/),
+and their equivalents in
+Splunk distribution of OpenTelemetry .NET.
+
+## Main settings
+
+The following settings are common to most instrumentation scenarios:
+
+| Setting                   | Description                                                                           | OpenTelemetry equivalent                       |
+|---------------------------|---------------------------------------------------------------------------------------|------------------------------------------------|
+| `SIGNALFX_ENV`            | The value for the `deployment.environment` tag added to every span.                   | Should be set using `OTEL_RESOURCE_ATTRIBUTES` |
+| `SIGNALFX_SERVICE_NAME`   | The name of the application or service.                                               | `OTEL_SERVICE_NAME`                            |
+| `SIGNALFX_VERSION`        | The version of the application. When set, it populates the `version` tag on spans.    | Should be set using `OTEL_RESOURCE_ATTRIBUTES` |
```

Comment:
```suggestion
| `SIGNALFX_VERSION`        | The version of the application. When set, it populates the `version` tag on spans.    |  Use `OTEL_RESOURCE_ATTRIBUTES` |
```

Reply:
Can we add an example for both?

Reply:
added in https://github.com/signalfx/splunk-otel-dotnet/pull/124/commits/ab38c08fb5198569a45b5296a35b1b9d9cbf0688